corehq/apps/hqadmin/utils.py

```diff
@@ -0,0 +1,19 @@
+from dimagi.utils.logging import notify_exception
+from pillowtop.utils import get_all_pillows
+
+from .models import PillowCheckpointSeqStore
+
+
+def pillow_seq_store():
+    for pillow in get_all_pillows():
+        checkpoint = pillow.get_checkpoint()
+        store, created = PillowCheckpointSeqStore.objects.get_or_create(checkpoint_id=checkpoint['_id'])
+        if not created and checkpoint['seq'] < store.checkpoint_id:
+            notify_exception(
```

Comment:
(you could convert the part before the hyphen to an int and do the check to workaround this)